cypress/support/api/requests/Customer.js

```diff
@@ -33,11 +33,13 @@ export function createCustomer(email, customerName, address, isActive = false) {
 }
 
 export function deleteCustomersStartsWith(startsWith) {
-  getCustomers(startsWith).then(resp => {
+  const serverStoredEmail = startsWith.toLowerCase();
```

Comment:
```suggestion
  const serverStoredEmail = email.toLowerCase();
```

**suggestion**: Rename `startsWith` argument to `email` or `customerEmail`

I think changing argument variable will help with readabilty.